plugins/poll/assets/javascripts/discourse/components/poll-options-dropdown.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="irv-dropdown">
```

Comment:
do you need the wrapping div here?

Reply:
good catch 👍 - it is indeed redundant.  Removed!